sources/assets/zsh/history.d/rustscan

```diff
@@ -0,0 +1,4 @@
+rustscan -a "$CIDR" --scripts custom --ulimit 5000
+rustscan -a "$CIDR" --greppable
+rustscan -a "$CIDR" -p 80,443,8080
+rustscan -a "$CIDR" --top
```

Comment:
Please use long format option
- `--addresses` instead of `-a`
- `--ports` instead of `-p`
Also please replace `$CIDR` with `$TARGET` (there is a set of defined arguments to be used dynamically in the history)

Reply:
This is done. Commits squashed.